src/helm/proxy/clients/together_client.py

```diff
@@ -12,6 +13,12 @@
 from .client import Client, wrap_request_time, truncate_sequence
 
 
+MODEL_ALIASES = {
+    "bloomz": "bloomz-176b-alpa",
```

Comment:
`-alpha`: Will there be future versions?

Reply:
According to Together: "the convention is `<model-name>-<size>-<framework>`"

There might be alternate implementations.

Reply:
If there will be alternate implementations, is it okay to cache results as plain `bloomz`?

Reply:
As discussed in the other thread, I changed this to cache results as the full name including the framework e.g. `bloomz-176b-alpa`.